designer/server/src/main/scala/pl/touk/nussknacker/ui/util/ApiVersion.scala

```diff
@@ -0,0 +1,7 @@
+package pl.touk.nussknacker.ui.util
+
+import derevo.circe._
+import derevo.derive
+
+@derive(encoder, decoder)
+final case class ApiVersion(value: Int)
```

Comment:
* it's used in `MigrationApiEndpoints` so it should be defined there
* should we add `@derive(schema)` as well?

Reply:
please take a look at my improvement suggestions: 
https://github.com/TouK/nussknacker/pull/5975